src/GraphQL.Tests/Builders/FieldBuilderTests.cs

```diff
@@ -357,30 +357,10 @@ public void getting_specified_argument_in_resolver_overrides_default_value()
         public void can_access_object()
         {
             var objectType = new ObjectGraphType();
-            objectType.Field<StringGraphType>()
-                .WithObject<int>()
-                .Resolve(context =>
-                {
-                    context.Object.ShouldEqual(12345);
-                    return null;
-                });
-
-            var field = objectType.Fields.First();
-            field.Resolve(new ResolveFieldContext
-            {
-                Source = 12345
-            });
-        }
-
-        [Fact]
-        public void can_access_object_with_custom_resolver()
-        {
-            var objectType = new ObjectGraphType();
-            objectType.Field<StringGraphType>()
-                .WithObject(obj => (int)obj + 1)
```

Comment:
this use case is now not possible, however It struck me that you can put whatever resolution logic you want in `Resolve` . Is there a use case for this I'm not thinking of?